cr-examples/triton/src/test/java/oracle/code/triton/TestMatrix.java

```diff
@@ -460,9 +466,9 @@ static void matmul_kernel(
             // The stride variables represent how much to increase the ptr by when moving by 1
             // element in a particular dimension. E.g. `stride_am` is how much to increase `a_ptr`
             // by to get the element one row down (A has M rows).
-            int stride_am, int stride_ak,
-            int stride_bk, int stride_bn,
-            int stride_cm, int stride_cn,
+            int stride_am, @Constant int stride_ak,
+            int stride_bk, @Constant int stride_bn,
+            int stride_cm, @Constant int stride_cn,
```

Comment:
Why the marking as constants?

Reply:
Triton will mark these three stride values as constants if they are equal to one at compile time. We mark them as constants directly as our code is not compiled by Triton.